OpenBSD revision 1.5
Remove a couple more bogus man pages and add missing MLINKs.  The
makewhatis info for the perl man pages now matches the pages themselves.

diff -x CVS -Nru ../perl-5.12.2/lib/AnyDBM_File.pm ./lib/AnyDBM_File.pm
--- ../perl-5.12.2/lib/AnyDBM_File.pm	Sun Sep  5 08:14:33 2010
+++ ./lib/AnyDBM_File.pm	Mon Oct 12 18:25:48 2009
@@ -19,8 +19,6 @@
 
 AnyDBM_File - provide framework for multiple DBMs
 
-NDBM_File, DB_File, GDBM_File, SDBM_File, ODBM_File - various DBM implementations
-
 =head1 SYNOPSIS
 
     use AnyDBM_File;
